### ISSUE TITLE
File open/save is missing on sql editor.

### ISSUE THREAD

**Issue by ayder** [beekeeper-studio/beekeeper-studio#28]
I would like to open sql files from filesystem as well as save them there.

**Comment by rathboma** [comment]
Yep! Totally makes sense :-).

We'll track the todo in this issue. Temporary workaround is to copy/paste from your sql file.

**Comment by iKlsR** [comment]
Worth noting that drag and drop works naturally for the textarea.

**Comment by gnovaro** [comment]
Adding a menu bar? For example if you want to establish another connection..  and other features like Formating SQL.

**Comment by rathboma** [comment]
Yep, that's on the cards too, but we're starting simple and working iteratively. There will be fairly regular releases with new features as we get to them :-). Just 2 of us with less than 5 hours a week each working on this really.

**Comment by iKlsR** [comment]
If you can draft up some milestones and put features under what's next for
a release I'm sure myself and others wouldn't mind jumping in. I've been
wanting something simple and lightweight like this for a while now.

**Comment by gnovaro** [comment]
Hi to all, why no use the "Projects" section? and maybe the community can contribute. I like the project, but can't replace my Sqlyog for now... I use a lot of features, like backup, restore, create index, history tab, export to csv/excel and change separator integrated in the UI.

**Comment by phillpafford** [comment]
Any update on this? I think this request is related https://github.com/beekeeper-studio/beekeeper-studio/issues/256

**Comment by rathboma** [comment]
Yes, they're duplicates really.

We're running a bit slow right now due to homeschooling and such, but hopefully we'll get to this soon.

**Comment by mariusa** [comment]
Ideally, when connecting to a saved connection, beekeeper should also open the sql files that were opened last time.

**Comment by marcosmacedoo** [comment]
> marcos

otário
